### ISSUE TITLE
use `make` to startup DouTok services

### ISSUE THREAD

**Issue by BaiZe1998** [TremblingV5/DouTok#112]
The local startup method for DouTok services is very poor, it needs to be optimized using make.

**Comment by TremblingV5** [comment]
Just proposal

We may need some `make ...` command to do following operations:

1. Use `make start all` to start all services in `./applications`
2. Use `make start xxx` to start a clarify service named `xxx` in `./applications/xxx`
3. Use `make stop all` to stop all services
4. Use `make stop xxx` to stop a clarify service

**Comment by BaiZe1998** [comment]
I think that the current method of launching all services by reading environment variables is not suitable, because modifying environment variables is not convenient. I would prefer to launch services by reading a local yml configuration file, or after the successful deployment of /env/dependences.yml, import the configuration of each service into etcd and get configuration from it.

**Comment by BaiZe1998** [comment]
@TremblingV5 cc

**Comment by BaiZe1998** [comment]
At present, the startup of services depends on environment variables and .yml files. I think it's better not to introduce too much complexity.

**Comment by TremblingV5** [comment]
> I think that the current method of launching all services by reading environment variables is not suitable, because modifying environment variables is not convenient. I would prefer to launch services by reading a local yml configuration file, or after the successful deployment of /env/dependences.yml, import the configuration of each service into etcd and get configuration from it.

Yes, I also feel not suitable in running DouTok.

Using env variables becuase I assumed that we may use k8s to manage services. If we use k8s, it's good for using env variables. But now we not using k8s so it's redundant.

I think we may use 2 - 3 launch layout:

1. Load configuration files
2. Load from etcd
3. Load env variables

DouTok can select a suitable method to load configurations. Now I think load files or load from etcd is better than env variables.

**Comment by AbigailJixiangyuyu** [comment]
May I directly comment the code related to Otel?
such as this one 
![image](https://github.com/TremblingV5/DouTok/assets/129409676/aa2e3e4b-52fa-442c-8884-29166587524e)

**Comment by TremblingV5** [comment]
> May I directly comment the code related to Otel? such as this one ![image](https://private-user-images.githubusercontent.com/129409676/309459612-aa2e3e4b-52fa-442c-8884-29166587524e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkzNjg1MTksIm5iZiI6MTcwOTM2ODIxOSwicGF0aCI6Ii8xMjk0MDk2NzYvMzA5NDU5NjEyLWFhMmUzZTRiLTUyZmEtNDQyYy04ODg0LTI5MTY2NTg3NTI0ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwMlQwODMwMTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMjA1ZmYxODA5YWI4NWJiNDQwODk2YTlhMThiYmVmMzcwYWIwYWE4ODRmNWI5NDllZjc1OGEyMTRlMDI2N2ZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.IJF-NxpAVCIuG7JmMXxlo6jpPbUG8cCrXuuB4YY5yyA)

Yes, but these code only log lines for can't achieve otel , it not affect run services.

**Comment by 011d** [comment]
hi, can I try to do this?

**Comment by TremblingV5** [comment]
> hi, can I try to do this?

Sure, but I prefer you to look this repo: https://github.com/cloudzenith/DouTok

It's the next version of this repo
